### PR TITLE
Implement setmodlog command

### PR DESCRIPTION
**Pull request by OTCompa**

### CODE REVIEW

File: internal/discord/commands.go [hunk 1]
```diff
@@ -110,3 +112,22 @@ func (d *Discord) Exile(s *discordgo.Session, i *discordgo.InteractionCreate) {
...
+// All logged commands will be logged to this channel.
+// Fields:
+//
+//	channel:	Channel
+func (d *Discord) SetModLog(s *discordgo.Session, i *discordgo.InteractionCreate) {
+	options := i.ApplicationCommandData().Options
+	channelID := options[0].ChannelValue(nil).ID
+	d.ModLogChannelID = channelID
+
+	tempstr := fmt.Sprintf("Mod log channel set to: <#%v>", channelID)
```
Comment (brtran4) [inline internal/discord/commands.go:126]: nit: `Mod logging channel...`

File: internal/discord/commands.go [hunk 2]
```diff
@@ -110,3 +112,22 @@ func (d *Discord) Exile(s *discordgo.Session, i *discordgo.InteractionCreate) {
...
+
+	err := StartInteraction(s, i.Interaction, tempstr)
+	if err != nil {
+		fmt.Printf("Unable to send ephemeral message: %v\n", err)
+	}
+	fmt.Printf("Set the moderation log channel to channel: %v\n", channelID)
```
Comment (brtran4) [inline internal/discord/commands.go:132]: nit: `Set the moderation logging channel...`

File: internal/discord/discord.go [hunk 3]
```diff
@@ -7,8 +7,9 @@ import (
 )
 
 type Discord struct {
-	Token   string
-	Session *discordgo.Session
+	Token           string
+	Session         *discordgo.Session
+	ModLogChannelID string
```
Comment (brtran4) [inline internal/discord/discord.go:12]: we dont have to do this now, but there may be a chance in the future we are implementing several logging channels for different things

Comment (brtran4) [reply internal/discord/discord.go:12]: (resolving btw no action required)

**Review by brtran4 — CHANGES_REQUESTED** [review]
nit: change PR message to `Implement setmodlog command`
also nit: let's change all vars to some variation of `...ModLogging...`

**Comment by OTCompa** [comment]
I messed something up resolving the merge conflict, will try to fix

**Review by brtran4 — APPROVED** [review]

**Pull request merged**